esmvaltool/cmor/_fixes/CMIP5/ACCESS1_0.py

```diff
@@ -0,0 +1,29 @@
+"""Fixes for ACCESS1-0 model"""
+from cf_units import Unit
+from ..fix import Fix
+
+
+class allvars(Fix):
+    """Common fixes to all vars"""
+
+    def fix_metadata(self, cubes):
+        """
+        Fix metadata
+
+        Fixes wrong calendar 'gregorian' instead of 'proleptic_gregorian'
+
+        Parameters
+        ----------
+        cube: iris.cube.Cube
+
+        Returns
+        -------
+        iris.cube.Cube
+
+        """
+        for cube in cubes:
+            coord_names = [coord.name() for coord in cube.coords()]
+            if 'time' in coord_names:
+                time = cube.coord('time', dim_coords=True)
+                time.units = Unit(time.units.name, 'gregorian')
```

Comment:
You can simplify this code by using

```python
try:
    time = cube.coord('time')
except iris.exceptions.CoordinateNotFoundError:
    continue

```

Reply:
Ok, I changed it and tested that it works with my recipe. Sorry for the  PEP8 errors. I don't find recipe_derive_test.yml, therefore I cannot test this?

Reply:
It is in `esmvaltoolrecipes/examples`

Reply:
In esmvaltool/recipes/examples (and esmvaltool/recipes) I don't find a recipe_derive_test.yml under version2_developement, in esmvaltool/recipes/example are:
        recipe_check_obs.yml
	recipe_concatenate_exps.yml
	recipe_correlation.yml 
	recipe_my_personal_diagnostic.yml
	recipe_ncl.yml
	recipe_preprocessor_derive_test.yml
	recipe_preprocessor_test.yml
	recipe_python.yml
	recipe_python_object_oriented.yml 
	recipe_r.yml 
	recipe_variable_groups.yml

Reply:
Probably recipe_preprocessor_derive_test.yml?